todoapp/codelab/25mar2017/05_todo_list_adapter.md

```diff
@@ -0,0 +1,30 @@
+
+[\[Paso Anterior\]](04_ic_pin_disabled.md)
+
+#ToDoListAdapter.java
+
+En nuestro `ToDoListAdapter.java` agregaremos la funcionalidad para que cuando nuestro ToDo no contenga location muestre nuestro `ic_pin_disabled.xml` que ayudara al usuario a entender que el ToDo no contiene información de location.
+
+```java
+ void bindData(ToDoContent toDoItem) {
+            item = toDoItem;
+            toDoNameTextBox.setText(toDoItem.getTitle());
+            if (toDoItem.hasDate()) {
+                dateTextBox.setText(toDoItem.getDate().toString());
+            }
+            doneCheckbox.setOnClickListener(this);
+            mapButton.setOnClickListener(this);
+            container.setOnClickListener(this);
+            if(item.getLat() == 0.0d && item.getLng() == 0.0d){
```

Comment:
again: this is entirely not true, what if I am actually in lat 0 and longitude 0?, logic error here, but it will work for this example, JFYI.